registries/_format/relative-json-pointer.md

```diff
@@ -0,0 +1,31 @@
+---
+owner: baywet
+issue:
+description: A JSON string representation of a relative JSON Pointer as defined in draft RFC 01
```

Comment:
I'd have some hesitation about encoding this format while it's still in a draft FWIW.

Reply:
@rattrayalex The status of Relative JSON Pointer is somewhat complicated.  I was talking with @darrelmiller about how to get that done a while back, which we should perhaps pick up again (not in this PR).

The point here is that JSON Schema (technically also a draft) defines the `relative-json-pointer` format so it needs to be in this table.

Reply:
To pile on to Henry's point.  The registry isn't trying to be normative in so far as what formats OpenAPI tooling must support.  It is trying to document what is out there being used, regardless of its status.  We have added a "deprecated" column to help guide people away from things that we think have better solutions today.

The objective of the registry is to be a place we can point authors of OpenAPI descriptions to in order to answer the question "What format should I use for this type?".  We hope this will help with the current situation of OpenAPI description authors doing things a bunch of different ways and tooling having to try and support all of them.

Reply:
Thank you for explaining! Makes total sense.